core/types/snapshot_nodes.go

```diff
@@ -1069,6 +1071,7 @@ func (p Payload) IntoProto() *snapshot.Payload {
 	case *snapshot.Payload_Teams:
 		ret.Data = dt
 	case *snapshot.Payload_TeamSwitches:
+	case *snapshot.Payload_Vesting:
 		ret.Data = dt
```

Comment:
Think we're now missing the `ret.Data = dt` for `Payload_TeamSwitches`.

Reply:
oh a terrible rebase again.

Reply:
addressed